spec/hand_rankings/hand_entities/high_card_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'poker_hands/hand_rankings/hand_entities/high_card'
+require 'poker_hands/card'
+require 'spec_helper'
+
+
+RSpec.describe PokerHands::Entities::HighCard do
+  subject { PokerHands::Entities::HighCard.new(cards: hand) }
+  let(:hand) do
+    [
+      PokerHands::Card.new(4, 'H'),
+      PokerHands::Card.new(11, 'S'),
+      PokerHands::Card.new(8, 'D'),
+      PokerHands::Card.new(2, 'S'),
+      PokerHands::Card.new(9, 'C')
+    ]
+  end
+
+  describe 'HighCard' do
+    it 'receives a poker hand' do
+      expect(PokerHands::Entities::HighCard).to receive(:new).with(cards: hand)
+      PokerHands::Entities::HighCard.new(cards: hand)
+    end
+  end
+
+  describe 'HighCard Entity' do
```

Comment:
Generally we use `describe` to indicate when we are testing a particular method on our class, which in this case we aren't. We can probably drop the describe here, and stick to having only the `it` block.